packages/database/src/changeHandlers/TaskCompletionHandler.js

```diff
@@ -42,7 +42,7 @@ export class TaskCompletionHandler extends ChangeHandler {
       })),
     );
 
-    const tasks = await this.models.task.find({
+    return this.models.task.find({
       // only fetch tasks that have a status of 'to_do'
       status: 'to_do',
```

Comment:
Just realised this status will need to be updated to also fetch when status is null, since repeating tasks will have a null status